service-src/service_snlua.c

```diff
@@ -186,10 +186,11 @@ timing_resume(lua_State *L, int co_index, int n) {
 	lua_State *co = lua_tothread(L, co_index);
 	lua_Number start_time = 0;
 	if (timing_enable(L, co_index, &start_time)) {
-		start_time = get_time();
 #ifdef DEBUG_LOG
+		double ti = diff_time(start_time);
```

Comment:
此处计算 diff_time 是一个没什么意义的时间值。 ti 输出若有意义，也是 start_time 这个时刻。

Reply:
那么大大，按你们思路来，我继续学习看代码。
我暂时的理解与同文件中的 lstop() 函数中也有下面这一段代码，所以我觉得应该是想求差值
```suggestion
		double ti = diff_time(start_time);
```

Reply:
那么应该改成这样子的意思是么，那我是继续提交，还是大大你们改就好了？
```suggestion
		fprintf(stderr, "PROFILE [%p] resume %lf\n", co, start_time);
```

Reply:
这三行代码没用，删掉就好了

Reply:
噢 那交给你们啦 我先学习~

Reply:
谢谢 这个可以关掉啦